code/modules/mob/living/carbon/alien/diona/diona_nymph.dm

```diff
@@ -38,6 +38,28 @@
 	var/dark_survival = 216                   // How long this diona can survive in darkness after energy is gone, before it dies
 	var/datum/dionastats/DS
 	var/mob/living/carbon/gestalt = null      // If set, then this nymph is inside a gestalt
+	var/kept_clean = 0
+
+	var/mob/living/carbon/alien/diona/master_nymph //nymph who owns this nymph if split. AI diona nymphs will follow this nymph, and these nymphs can be controlled by the master.
+	var/list/mob/living/carbon/alien/diona/birds_of_feather = list() //list of all related nymphs
```

Comment:
This should honestly use softrefs or whatever the other one is. Hard references to mobs are bad. Check the wiki.